api/websocket/server/relay.go

```diff
@@ -49,10 +49,30 @@ func (ws *WsServer) sendOutboundRelayMessage(srcAddrStrPtr *string, msg *pb.Outb
 		return
 	}
 
+	payloads := msg.Payloads
```

Comment:
Is "sendOutboundRelayMessage" called by client?

Reply:
It's called in `api/websocket/server/server.go`